src/styles/main.scss

```diff
@@ -1,3 +1,262 @@
+  height: 4px;
+
+  background-color: $blue-accent;
+  border-radius: 8px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+
+  border: 1px solid $gray-elements;
+  border-radius: 5px;
+
+  font-size: 10px;
+  line-height: 14px;
+  padding: 32px 16px 16px;
+
+  color: $main-accent;
+  background-color: $white;
+
+  margin: 0;
+
+  transition: transform 300ms ease-in-out;
+
+  &__image {
+    margin: 0;
+    display: flex;
+    justify-content: center;
+    margin-bottom: 40px;
+
+    &-img {
+      width: 160px;
+      height: 134px;
+    }
+  }
+
+  &__product-title {
+    margin: 0 0 4px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: $big_line-height;
+  }
+
+  &__product-code {
+    margin: 0 0 16px;
+    color: $secondary;
+  }
+
+  &__review {
+    margin: 0;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__review-text {
+    margin: 0;
+    color: $main-accent;
+    padding: 0;
+  }
+
+  &__price {
+    margin: 0;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    &-value {
+      margin: 0;
+      font-weight: 700;
+      font-size: 16px;
+      line-height: $big_line-height;
+    }
+
+    &-text {
+      margin: 0;
+      font-size: 12px;
+      line-height: $big_line-height;
+    }
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    background-color: $blue-accent;
+    border: 1px solid $blue-accent;
+
+    text-transform: uppercase;
+    text-decoration: none;
+    padding: 12px 70px;
+
+    font-weight: 700;
+    font-size: 14px;
+    border-radius: 5px;
+    line-height: 16px;
+    color: $white;
+
+    transition: 300ms ease-in-out;
+  }
+
+  &__link:hover {
+    background-color: $white;
+    border: 1px solid $blue-accent;
+    color: $blue-accent;
+  }
+
+  &:hover {
+    cursor: pointer;
+    transform: scale(1.1);
+  }
+
+  &:hover &__product-title {
+    color: #34568b;
+  }
+}
+
+.stars {
+  display: flex;
+
+  &--1 :first-child,
+  &--2 :nth-child(-n+2),
+  &--3 :nth-child(-n+3),
+  &--4 :nth-child(-n+4),
+  &--5 :nth-child(-n+5) {
+    background: url(../images/star-active.svg) no-repeat;
+  }
+
+  &__star {
+    background: url(../images/star.svg) no-repeat 50%;
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+  }
+}
+
+.cards-container {
+  display: grid;
+  padding-top: 50px 0 72px;
+  grid-template-columns: repeat(var(--column-count),200px);
+  row-gap: 46px;
+  column-gap: 48px;
+  justify-content: center;
+
+  --column-count: 1;
+}
+
+@media (max-width: 488px) {
+  .cards-container {
+    --column-count: 1;
+  }
+}
+
+@media (min-width: 488px) {
+  .cards-container {
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .cards-container {
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .cards-container {
+    --column-count: 4;
+  }
```

Comment:
It's redundant for this task 
![image](https://github.com/mate-academy/layout_catalog/assets/70009205/9df821ec-936e-41a0-92a4-bc2a60939cc8)